### ISSUE TITLE
Error in Install screen when importing saved selections

### ISSUE THREAD

**Issue by dmadsen** [ChrisTitusTech/winutil#2709]
## Describe the bug
Use Import on the Install screen; it displays an error message in the progress window.

## To Reproduce
Go to Install screen.
Click on gear.
Use “Import” to import a previously-exported json file.  It happened the first time on a saved file from a previous version of the software, but it seems to do the same thing on a file previously exported from the same version.

## Expected behavior
I expected the file to be imported without an error message.

## Screenshots (copied error message text)
```
Invoke-WPFPresets : Cannot process argument transformation on parameter 'preset'. Cannot convert value to type
System.String.
At line:4398 char:35
+         Invoke-WPFPresets -preset $flattenedJson -imported $true
+                                   ~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Invoke-WPFPresets], ParameterBindingArgumentTransformationException
    + FullyQualifiedErrorId : ParameterArgumentTransformationError,Invoke-WPFPresets
```
## Additional context
Hope this is a good enough report to help you figure out the problem!

**Comment by Cryostrixx** [comment]
Hello, @dmadsen!

I resolved this issue about a week ago in #2658, and the fix is included in the latest pre-release.

To update to the latest version of the script:
1. Open your Start menu.
2. Search for Terminal or PowerShell (whichever you prefer).
3. Right-click and select "Run as administrator."
4. Enter the following command: `irm christitus.com/windev | iex` and press Enter. This will run the latest pre-release.

-----

If you encounter issues with that method, you can download the latest pre-release directly from [here](https://github.com/ChrisTitusTech/winutil/releases/download/24.09.10/winutil.ps1).

To run the downloaded script:
1. Save the file to your Downloads folder.
2. Open Terminal or PowerShell as administrator.
3. Run these commands:
   * `sl $env:USERPROFILE\Downloads`
   * `.\winutil.ps1`
4. If prompted with a warning about an untrusted publisher, type `R` and press Enter to proceed.

-----

You should now have the latest pre-release running. I hope this helps!

**Comment by MyDrift-user** [comment]
Thanks for the report, this was fixed in #2658, but not mapped to this issue so it did not get closed.

I am closing this Issue now

/close
